Makefile

```diff
@@ -1,7 +1,7 @@
 
 # Image URL to use all building/pushing image targets
-IMG ?= goharbor/harbor-operator:dev
-RELEASE_VERSION ?= 0.0.0-dev
+IMG ?= goharbor/harbor-operator:v1.0.0
+RELEASE_VERSION ?= v1.0.0
```

Comment:
`RELEASE_VERSION ?= v1.0.0` => `RELEASE_VERSION ?= 1.0.0` , `v` prefix not needed.